actors/miner/src/notifications.rs

```diff
@@ -0,0 +1,182 @@
+use crate::{
+    PieceActivationManifest, PieceChange, SectorChanges, SectorContentChangedParams,
+    SectorContentChangedReturn, SECTOR_CONTENT_CHANGED,
+};
+use fil_actors_runtime::runtime::Runtime;
+use fil_actors_runtime::{
+    actor_error, ActorError, AsActorError, SendError, STORAGE_MARKET_ACTOR_ADDR,
+};
+use fvm_ipld_encoding::ipld_block::IpldBlock;
+
+use fvm_shared::address::Address;
+use fvm_shared::clock::ChainEpoch;
+use fvm_shared::econ::TokenAmount;
+use fvm_shared::sector::SectorNumber;
+use fvm_shared::sys::SendFlags;
+use num_traits::Zero;
+use std::collections::BTreeMap;
+
+pub struct ActivationNotifications<'a> {
+    pub sector_number: SectorNumber,
+    pub sector_expiration: ChainEpoch,
+    pub pieces: &'a Vec<PieceActivationManifest>,
+}
+
+/// Sends notifications of sector and piece activation to nominated receiving actors.
+/// Inputs are per-sector, matching parameters provided by external callers.
+/// This method groups inputs by receiving actor and sends a single notification per actor.
+/// Notifications are fire-and-forget, so there is no return value.
+/// If require_success is true, this method will return an error if any of the notifications fail
+/// or are rejected by the receiving actor.
```

Comment:
I had to Google to understand what you mean here. I have never used this command, but I interpret that it's a request to re-wrap the lines of this comment so that they fill the allowed column width? If so I respectfully decline, and prefer the current wrapping of breaking at sentence and phrase boundaries.